auth/tests/acc/auth-sign-in-journey.test.ts

```diff
@@ -0,0 +1,468 @@
+import dotenv from "dotenv";
+// Systems Manager Client
+import { SSMClient, GetParametersCommand } from "@aws-sdk/client-ssm";
+// Secrets Manager Client
+import {
+  GetSecretValueCommand,
+  SecretsManagerClient,
+} from "@aws-sdk/client-secrets-manager";
+
+dotenv.config();
+
+type METHOD_TYPE = "GET" | "POST";
+type HTTP_RESPONSE = Partial<IncomingMessage> & {
+  headers: IncomingHttpHeaders;
+  body: string;
+};
+
+/**
+ * Parameterisation
+ */
+const region = process.env["REGION"] ?? "eu-west-2";
+
+// Get Password
+const smClient = new SecretsManagerClient({ region: region });
+const passwordResponse = await smClient.send(
+  new GetSecretValueCommand({
+    SecretId: "PASSWORD",
+  }),
+);
+const password = passwordResponse.secretString;
+
+const secretResponse = await smClient.send(
+  new GetSecretValueCommand({
+    SecretId: "SECRET",
+  }),
+);
+const secret = secretResponse.secretString;
+
+// Get parameters from Systems Manager
+const input = {
+  Names: [
+    "PROXY_DOMAIN",
+    "COGNITO_DOMAIN",
+    "COGNITO_APP_CLIENT_ID",
+    "COGNITO_APP_CLIENT_REDIRECT_URL",
+    "ONELOGIN_STAGING_DOMAIN",
+    "REDIRECT_URI",
+    "EMAIL_ADDRESS",
+    "SCOPE",
+  ],
+  WithDecryption: true || false,
+};
+
+const command = new GetParametersCommand(input);
+const smResponse = await client.send(command);
+
+const PROXY_DOMAIN = smResponse.filter((p) => p.name === "PROXY_DOMAIN")[0]
+  .value;
+const COGNITO_DOMAIN = smResponse.filter((p) => p.name === "COGNITO_DOMAIN")[0]
+  .value;
+const ONELOGIN_OIDC_STAGING_DOMAIN =
+  smResponse.filter((p) => p.name === "ONELOGIN_OIDC_STAGING_DOMAIN")[0]
+    .value || "oidc.staging.account.gov.uk";
+const ONELOGIN_STAGING_DOMAIN =
+  smResponse.filter((p) => p.name === "ONELOGIN_STAGING_DOMAIN")[0].value ||
+  "staging.account.gov.uk";
+const COGNITO_APP_CLIENT_ID = smResponse.filter(
+  (p) => p.name === "COGNITO_APP_CLIENT_ID",
+)[0].value;
+const COGNITO_APP_CLIENT_REDIRECT_URL = smResponse.filter(
+  (p) => p.name === "COGNITO_APP_CLIENT_REDIRECT_URL",
+)[0].value;
+const REDIRECT_URI =
+  smResponse.filter((p) => p.name === "REDIRECT_URI")[0].value ||
+  "govuk://govuk/login-auth-callback";
+const SCOPE =
+  smResponse.filter((p) => p.name === "SCOPE")[0].value || "openid+email";
+const emailAddress = smResponse.filter((p) => p.name === "EMAIL_ADDRESS")[0]
+  .value;
+
+const redirectURL = `https://${ONELOGIN_STAGING_DOMAIN}/authorize?client_id=${COGNITO_APP_CLIENT_ID}&redirect_uri=https%3A%2F%2F${COGNITO_DOMAIN}%2Foauth2%2Fidpresponse&scope=${SCOPE}&response_type=code&state=H4sIAAAAAAAAAE2RzZKbMBCE30VnwxoEGHzLLhhDgrGzrFlIpVwySPxJiBgwhlTePfIllVvP9NczUzW_AQJbgEdpwv0gqRfi-cZO1dMSrMBVOLzFlBdVK8pMlIqqEF2plV5rWGNVljmy9VVnXfnkcwGUw9D125eX3NSoMpud1uRVJmeUjzm58XaQW
+zwIFAs04zkWkggZevabkAXY_gC8w22VPxmGKgp-rkAliICdyzDypiA-wcOSPYKleYRuMofuaUlrWh9YoCRxApP6QA92WQa280iYt07qAgZRyoQ3HZaShXYzhVEyp65IFGJLLYa_q7ohZPO8pLtvir17vjumYfl-Ve-U-wI36fD1VxR3xwv2hpBZia7eR0RnstPamufIxE5c4ta_KfHF_
```

Comment:
Is it secure to be committing the `state` value to git? Or is it no longer valid?